composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "thg/thg-client",
     "description": "THG Open API client",
     "type": "library",
-    "version": "1.3.0",
+    "version": "1.3.1",
```

Comment:
@andy-lu-42 You need to bump the version whenever changes are made to repository